packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-rds-walkthrough.ts

```diff
@@ -1,3 +1,4 @@
+import * as path from 'node:path'
```

Comment:
you could probably just use `import * as path from 'path';` as seen in [other places in repo](https://github.com/aws-amplify/amplify-category-api/blob/main/packages/amplify-category-api/src/commands/api/add-graphql-datasource.ts#L15).

Reply:
I will change this, force of habit. And it does not appear to be supported with Node 14